presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -43,28 +44,47 @@ fun AgendaScreen(
     agendaScreenEvent: (AgendaScreenEvent) -> Unit,
     onSelectedEditAgendaItemClicked: (id: String, AgendaType, agendaMenuActionType: AgendaMenuActionType) -> Unit,
     onSelectedAgendaItem: (agendaType: Int) -> Unit, /* TODO Check where this is being used */
+    onLogout: () -> Unit,
     modifier: Modifier = Modifier) {
 
     val calendarState = rememberUseCaseState()
 
     Scaffold(
         modifier = modifier,
         topBar = {
-            AgendaTopBar(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .background(color = MaterialTheme.colorScheme.backgroundBackColor)
-                    .padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
-                initials = agendaScreenState.usersInitials,
-                displayMonth = agendaScreenState.selectedDate.month.toString(),
-                onProfileButtonClicked = {
-                    /** TODO Open dropdown menu here */
-                    Log.d("AGENDA_SCREEN", "Profile button clicked")
-                },
-                onDateClicked = {
-                    calendarState.show()
-                },
-            )
+            Column(modifier = Modifier.fillMaxWidth()) {
+                AgendaTopBar(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .background(color = MaterialTheme.colorScheme.backgroundBackColor)
+                        .padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
+                    initials = agendaScreenState.usersInitials,
+                    displayMonth = agendaScreenState.selectedDate.month.toString(),
+                    onProfileButtonClicked = {
+                        agendaScreenEvent(AgendaScreenEvent.OnOpenLogoutDropDownMenu(shouldOpen = true))
+                    },
+                    onDateClicked = {
+                        calendarState.show()
+                    },
+                )
+
+                AgendaDropDownMenu(
+                    modifier = Modifier
+                        .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor)
+                        .align(Alignment.End),
+                    shouldOpenDropdown = agendaScreenState.shouldOpenLogoutDropDownMenu,
+                    onCloseDropdown = {
+                        agendaScreenEvent(
+                            AgendaScreenEvent.OnChangedShowDropdownStatus(shouldOpen = false)
+                        )
+                    },
+                    listOfMenuItemId = listOf(me.androidbox.presentation.R.string.logout),
```

Comment:
Why do you need a list tho?
Can´t it just be one id?

Reply:
Also I'd have some other kind of identifier for the items, otherwise you'll always have to remember the exact string (which can change)